CHANGELOG.md

```diff
@@ -5,6 +5,40 @@
 ### Added
 
 - Added metric for throttled block device events.
+- Added metrics for counting rate limiter throttling events.
+- Added metric for counting MAC address updates.
+- Added metrics for counting TAP read and write errors.
+- Added metrics for counting RX and TX partial writes.
+- Added metrics that measure the duration of pausing and resuming the microVM,
+  from the VMM perspective.
+- Added metric for measuring the duration of the last full snapshot created,
+  from the VMM perspective.
+- Added metric for measuring the duration of loading a snapshot, from the VMM
+  perspective.
+- Added metrics that measure the duration of pausing and resuming the microVM,
+  from the API (user) perspective.
+- Added metric for measuring the duration of the last full snapshot created,
+  from the API (user) perspective.
+- Added metric for measuring the duration of loading a snapshot, from the API
+  (user) perspective.
+
+### Fixed
+
+- Boot time on AMD achieves the desired performance (i.e under 150ms).
+
+### Changed
+
+- The logger `level` field is now case-insensitive.
+- Disabled boot timer device after restoring a snapshot.
+- Enabled boot timer device only when specifically requested, by using the
+  `--boot-timer` dedicated cmdline parameter.
+- `firecracker/jailer --version` now gets updated on each devtool
+  build to the output of `git describe --dirty`, if the git repo is available.
+
+## [0.22.0]
```

Comment:
Can't comment on the exact line 😢 :

```
- Added `track_dirty_pages` field to `machine-config`.
- Added a new API call, `PUT /snapshot/create`, for creating a full snapshot.
- Added a new API call, `PUT /snapshot/load`, for loading a snapshot.
```

(and possibly more) were not released as part of `v0.22.0`, they need to be moved to `Unreleased` as they will be available starting with `v0.23.0`

Reply:
Right. Some of the changes are not necessarily ordered it seems.